timeline/templates/timeline/timeline.html

```diff
@@ -36,17 +40,19 @@ <h3><i class="iconify" data-icon="mdi:clock-time-eight"></i> {{ timeline_title }
 <div class="container-fluid sodar-page-container">
   {% if object_list.count > 0 %}
     <div class="card mb-3" id="sodar-tl-event-list">
-      <div class="card-body p-0">
-        <table class="table table-striped sodar-card-table" id="sodar-tl-table">
-          <thead>
-            {% include 'timeline/_list_header.html' %}
-          </thead>
-          <tbody>
-            {% for event in object_list %}
-              {% include 'timeline/_list_item.html' %}
-            {% endfor %}
-          </tbody>
-        </table>
+      {% include 'timeline/_sort_header.html' with search_title=timeline_title %}
+        <div class="card-body p-0 sodar-sort-card-body">
+          <table class="table table-striped sodar-card-table sodar-timeline-table" id="sodar-tl-table">
+            <thead>
+              {% include 'timeline/_list_header.html' %}
+            </thead>
+            <tbody>
+              {% for event in object_list %}
+                {% include 'timeline/_list_item.html' %}
+              {% endfor %}
+            </tbody>
+          </table>
+        </div>
       </div>
     </div>
     {% if is_paginated %}
```

Comment:
We don't need to include the server-side pagination template anymore if we're doing it on the client side.